### DISCUSSION TITLE
Auth token

### DISCUSSION THREAD

**Discussion by Lasto13** [verdaccio/verdaccio#3971] [Q&A]
Hi, we are using Verdaccio deployed on ec2 instance with config bellow. We host our privare packages there but there is one package for which I want Verdaccio to act as a proxy. Currently every colleague has in his .npmrc config poiting to our registry and for desired registry for which we need to be authorized like: 

`registry=https://npm.OUR_DOMAIN.com/
@DESIRED:registry=https://npm.DESIRED.com/
//npm.DESIRED.com/:_authToken=00000000-0000-0000-0000-000000000000`

`

uplinks:
  DESIRED:
    url: https://npm.DESIRED.com/
    auth:
      type: bearer
      token: "00000000-0000-0000-0000-000000000000"
  npmjs:
    url: https://registry.npmjs.org/
packages:
  '@DESIRED/**':
    access: $all
    proxy: DESIRED
  '@*/*':
    access: $all
    publish: $authenticated
    proxy: npmjs
  '**':
    access: $all
    proxy: npmjs
`
![Screenshot 2023-08-14 at 18 36 37](https://github.com/verdaccio/verdaccio/assets/10142085/60c57612-23bf-4f19-a6ca-0c1c667d8f0e)

But When I install packages from DESIRED scope I get 401 with a message 
`3380 verbose stack HttpErrorAuthUnknown: Unable to authenticate, need: Basic realm="https://npm.DESIRED.com/",service="npm.DESIRED.com"` 

Even though no logs of 401 can be found on verdaccio logs, only in the logs on my local machine when it fails. but we are sure everything goes through our verdaccio instance. What can be the problem ? It seems like the verdaccio is not passing auth token allong (I also tried the configuration with adding auth headers)
When I try to manually
`curl -H 'Authorization: Bearer 00000000-0000-0000-0000-000000000000' https://npm.DESIRED.com/`
everything works as expected, from local and from verdaccio instance, even from within docker container

**Comment by juanpicado** [comment]
could you format code please ? 🙏🏼 (from mobile app is hard to read)

**Reply by Lasto13** [reply]
Hey Juan, Im sorry but this editor formats it badly so I added a screenshot.

**Reply by Lasto13** [reply]
In the screenshot there is authorization headers, but I also tried with type: bearer and token: xxx-xxx

**Reply by Lasto13** [reply]
Hello @juanpicado  :) Did you stumble upon this ticket again by any chance ? :)
